Makefile

```diff
@@ -1,23 +1,16 @@
-UGLIFY = node_modules/.bin/uglifyjs
```

Comment:
I love makefiles, and use them in all my projects. I don't contribute to Ramda much any longer, though, so I'm happy for you people to decide what is best for the project.

I urge you to keep in mind, @kedashoe, that as you currently seem to be the primary maintainer of this project you will be most affected by the consequences—positive and negative—of any changes to the way Ramda is built and distributed.